nifi-mock/src/main/java/org/apache/nifi/util/MockControllerServiceLookup.java

```diff
@@ -97,4 +97,5 @@ public String getControllerServiceName(final String serviceIdentifier) {
         final ControllerServiceConfiguration status = controllerServiceMap.get(serviceIdentifier);
         return status == null ? null : serviceIdentifier;
     }
+
```

Comment:
nit-pick: this is the only change to this file (line at the end). Can it be removed?